src/alphabet.rs

```diff
@@ -23,3 +23,112 @@ pub const FLICKR: &[u8; 58] = b"123456789abcdefghijkmnopqrstuvwxyzABCDEFGHJKLMNP
 /// The default alphabet used if none is given. Currently is the
 /// [`BITCOIN`](constant.BITCOIN.html) alphabet.
 pub const DEFAULT: &[u8; 58] = BITCOIN;
+
+/// Prepared Alpabet for [`EncodeBuilder`](crate::encode::EncodeBuilder) and [`DecodeBuilder`](crate::decode::DecodeBuilder).
+#[derive(Clone, Copy)]
+#[allow(missing_debug_implementations)]
+#[allow(missing_docs)]
+pub struct Alphabet {
+    pub encode: [u8; 58],
```

Comment:
These fields can be `pub(crate)`, with `const fn new` I don't think there's any reason users would need direct access.

Reply:
Agree with this. I thought about private field, but it was not possible with current way for creating predefined objects. Forgot about `pub(crate`).